private/rules/kt_jvm_export.bzl

```diff
@@ -70,11 +70,8 @@ def kt_jvm_export(
 
     javadocopts = kwargs.pop("javadocopts", None)
 
-    # ensure that the kotlin-stdlib is included in deploy_env
-    if KOTLIN_STDLIB not in deploy_env:
-        updated_deploy_env = deploy_env + [KOTLIN_STDLIB]
-    else:
-        updated_deploy_env = deploy_env
+    updated_excluded_workspaces = {name: None for name in excluded_workspaces}
```

Comment:
`deploy_env` gets added back into the docs so we were generating docs for kotlin stdlib which seems wrong. Changed to excluded_workspace to keep it out of the docs, which seems like what was actually intended.
This is also how we handle `scala_export` in confluent's fork of rules_jvm_external.